lib/tasks/rake_arguments.rb

```diff
@@ -0,0 +1,9 @@
+class RakeArguments
+  def self.task_options
```

Comment:
This a case of the name causing one to think in misleading directions.
There is nothing rake specific about the arguments, other than that they happen to be arguments to the rake command.